packages/cascara/src/ui/Table/TableStyleTest.fixure.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import faker from 'faker';
-import './TableStyleTest.module.scss';
+//import './TableStyleTest.module.scss';
```

Comment:
@lordpixel was this intentional disabling the style import in the table style fixture?